AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/MutationSync/OutgoingMutationQueue/SyncMutationToCloudOperation.swift

```diff
@@ -179,6 +181,17 @@ class SyncMutationToCloudOperation: Operation {
         }
     }
 
+    private func resolveReachabilityPublisher(request: GraphQLRequest<MutationSync<AnyModel>>) {
+        if networkReachabilityPublisher == nil {
+            guard let apiName = request.apiName else {
+                return
+            }
+            //TODO: Determine where allowsCellularAccess is controlled
```

Comment:
As above, we don't expose the `allowsCellularAccess` flag yet.

Reply:
done.